addons/path-mapped/common/src/main/java/org/commonjava/indy/pathmapped/common/PathMappedController.java

```diff
@@ -70,6 +74,21 @@ public PathMappedListResult list( String packageType, String type, String name,
         {
             list = fileManager.list( storeKey.toString(), path );
         }
+        // filter by fileType
+        if ( fileType != null && list != null )
+        {
+            Arrays.stream( list ).map( s -> {
+                if ( "all".equals( fileType ) || ( "dir".equals( fileType ) && s.endsWith( "/" ) ) || (
+                                "file".equals( fileType ) && !s.endsWith( "/" ) ) )
```

Comment:
Seems this is also a filter, so why not just use filter(s->"all".equals( fileType ) || ( "dir".equals( fileType ) && s.endsWith( "/" ) ) || ( "file".equals( fileType ) && !s.endsWith( "/" ) )) instead of map(...)?

Reply:
Exactly right. I will update it.

Reply:
Updated.